metrics-jmx/src/main/java/com/codahale/metrics/jmx/JmxReporter.java

```diff
@@ -596,6 +598,7 @@ public void onHistogramAdded(String name, Histogram histogram) {
             } catch (JMException e) {
                 LOGGER.warn("Unable to register histogram", e);
             }
+            return null;
```

Comment:
Why don't we return the `Histogram` which has been passed into the method when it was added?